packages/diagrams/lib/src/cupertino_colors.dart

```diff
@@ -0,0 +1,688 @@
+  <String, Color>{'black': CupertinoColors.black},
+  <String, Color>{'lightBackgroundGray': CupertinoColors.lightBackgroundGray},
+  <String, Color>{
+    'extraLightBackgroundGray': CupertinoColors.extraLightBackgroundGray
+  },
+  <String, Color>{'darkBackgroundGray': CupertinoColors.darkBackgroundGray},
+  <String, Color>{'inactiveGray': CupertinoColors.inactiveGray},
+  <String, Color>{'destructiveRed': CupertinoColors.destructiveRed},
+];
+List<Map<String, CupertinoDynamicColor>> activeColors =
+    <Map<String, CupertinoDynamicColor>>[
+  <String, CupertinoDynamicColor>{'activeBlue': CupertinoColors.activeBlue},
+  <String, CupertinoDynamicColor>{'activeGreen': CupertinoColors.activeGreen},
+  <String, CupertinoDynamicColor>{'activeOrange': CupertinoColors.activeOrange},
+];
+List<Map<String, CupertinoDynamicColor>> systemColors1 =
+    <Map<String, CupertinoDynamicColor>>[
+  <String, CupertinoDynamicColor>{'systemBlue': CupertinoColors.systemBlue},
+  <String, CupertinoDynamicColor>{'systemGreen': CupertinoColors.systemGreen},
+  <String, CupertinoDynamicColor>{'systemIndigo': CupertinoColors.systemIndigo},
+  <String, CupertinoDynamicColor>{'systemOrange': CupertinoColors.systemOrange},
+  <String, CupertinoDynamicColor>{'systemPink': CupertinoColors.systemPink},
+  <String, CupertinoDynamicColor>{'systemPurple': CupertinoColors.systemPurple},
+  <String, CupertinoDynamicColor>{'systemRed': CupertinoColors.systemRed},
+  <String, CupertinoDynamicColor>{'systemTeal': CupertinoColors.systemTeal},
+  <String, CupertinoDynamicColor>{'systemYellow': CupertinoColors.systemYellow},
+];
+List<Map<String, CupertinoDynamicColor>> systemColors2 =
+    <Map<String, CupertinoDynamicColor>>[
+  <String, CupertinoDynamicColor>{'systemGrey': CupertinoColors.systemGrey},
+  <String, CupertinoDynamicColor>{'systemGrey2': CupertinoColors.systemGrey2},
+  <String, CupertinoDynamicColor>{'systemGrey3': CupertinoColors.systemGrey3},
+  <String, CupertinoDynamicColor>{'systemGrey4': CupertinoColors.systemGrey4},
+  <String, CupertinoDynamicColor>{'systemGrey5': CupertinoColors.systemGrey5},
+  <String, CupertinoDynamicColor>{'systemGrey6': CupertinoColors.systemGrey6},
+];
+List<Map<String, CupertinoDynamicColor>> labelColors =
+    <Map<String, CupertinoDynamicColor>>[
+  <String, CupertinoDynamicColor>{'label': CupertinoColors.label},
+  <String, CupertinoDynamicColor>{
+    'secondaryLabel': CupertinoColors.secondaryLabel
+  },
+  <String, CupertinoDynamicColor>{
+    'tertiaryLabel': CupertinoColors.tertiaryLabel
+  },
+  <String, CupertinoDynamicColor>{
+    'quaternaryLabel': CupertinoColors.quaternaryLabel
+  },
+  <String, CupertinoDynamicColor>{
+    'secondarySystemFill': CupertinoColors.secondarySystemFill
+  },
+  <String, CupertinoDynamicColor>{
+    'tertiarySystemFill': CupertinoColors.tertiarySystemFill
+  },
+  <String, CupertinoDynamicColor>{
+    'quaternarySystemFill': CupertinoColors.quaternarySystemFill
+  },
```

Comment:
Missing `systemFill`, and these could probably go with the other system colors

Reply:
Thanks!